### PR TITLE
office address changes

### PR DESCRIPTION
**Pull request by detaybey**
footer & sub pages address change

### CODE REVIEW

**Review by savas-guven — APPROVED** [review]
blah

File: _includes/footer.html [hunk 1]
```diff
@@ -62,9 +62,9 @@ <h3 class="module-title ">
                             <div class="custom footerlocation">
                                 <svg class="footericon"><use xlink:href="../images/sprite.svg#icon-map-o"></use></svg>
                                 <br />
-                                <a href="https://www.google.com/maps/place/Landsberger+Str.+302,+80687+München,+Germany/@48.14355,11.4984984,17z/data=!3m1!4b1!4m5!3m4!1s0x479e762aeaf2d807:0xc5a85816304c8a87!8m2!3d48.14355!4d11.5006871"
-                                    target="_blank" class="footermail">Landsbergerstr. 302<br />
-                                    D-80687 München<br />
+                                <a href="https://www.google.com/maps/place/Zeppelinstra%C3%9Fe+73,+81669+M%C3%BCnchen,+Germany/@48.1301598,11.5842197,17z/data=!3m1!4b1!4m5!3m4!1s0x479ddf7cc4143749:0x1d94ae4c5d1f2e69!8m2!3d48.1301562!4d11.5864084"
```
Comment (v1r7u) [inline _includes/footer.html:65]: Are ß and ü intentionally escaped?

File: contact-us.md [hunk 2]
```diff
@@ -70,8 +70,8 @@ layout: default
                         	<div class="col-md-3">
                                 <svg class="contacticon"><use xlink:href="../images/sprite.svg#icon-map-o"></use></svg>
                                 <br>
-                                <a href="https://www.google.com/maps/place/Landsberger+Str.+302,+80687+München,+Germany/@48.14355,11.4984984,17z/data=!3m1!4b1!4m5!3m4!1s0x479e762aeaf2d807:0xc5a85816304c8a87!8m2!3d48.14355!4d11.5006871" target="_blank">Landsbergerstr. 302<br>
-                                D-80687 München<br>
+                                <a href="https://www.google.com/maps/place/Zeppelinstra%C3%9Fe+73,+81669+M%C3%BCnchen,+Germany/@48.1301598,11.5842197,17z/data=!3m1!4b1!4m5!3m4!1s0x479ddf7cc4143749:0x1d94ae4c5d1f2e69!8m2!3d48.1301562!4d11.5864084" target="_blank">Zeppelinstr. 73<br>
```
Comment (v1r7u) [inline contact-us.md:73]: The same question here

**Review by v1r7u — COMMENTED** [review]
Escaped characters in two places. Is it ok?

**Comment by savas-guven** [comment]
The link seems to work

File: _includes/footer.html (hunk 1 above)
Comment (detaybey) [reply _includes/footer.html:65]: I just took the url from google maps. What's wrong?

File: contact-us.md (hunk 2 above)
Comment (detaybey) [reply contact-us.md:73]: Same

**Review by v1r7u — APPROVED** [review]

File: _includes/footer.html (hunk 1 above)
Comment (v1r7u) [reply _includes/footer.html:65]: I'm not saying it's wrong. In the previous version, we had `München`, in the new one - `M%C3%BCnchen`. The same word is written in different ways, which catches attention. That's it ;)

**Pull request merged**